src/wp-includes/block-template-utils.php

```diff
@@ -475,12 +475,15 @@ function _flatten_blocks( &$blocks ) {
  * stylesheet as a theme attribute into each wp_template_part
  *
  * @since 5.9.0
+ * @deprecated 6.4.0 Use _inject_theme_attribute_in_template_part_block() with traverse_and_serialize_blocks() instead.
  * @access private
  *
  * @param string $template_content serialized wp_template content.
  * @return string Updated 'wp_template' content.
  */
 function _inject_theme_attribute_in_block_template_content( $template_content ) {
+	_deprecated_function( __FUNCTION__, '6.4.0' );
```

Comment:
I think you can also put the message with the new recommendation as 3rd argument in the call.

Reply:
https://github.com/WordPress/wordpress-develop/pull/5307/commits/5572a1d71957c109e4644909ed3551006acc08bd